kubespawner/objects.py

```diff
@@ -615,8 +631,10 @@ def make_secret(
         going to be created in.
     username:
         The name of the user notebook.
-    ssl_directory:
+    cert_paths:
         Path to a directory containing all users certificates and keys.
```

Comment:
This is later used as a dictionary, but I understand it to be a string from this docstring.

Reply:
yes, I left the old doc string on accident good catch.